abdiff/core/run_ab_transforms.py

```diff
@@ -113,7 +146,7 @@ def run_docker_container(
             "source": source,
             "input_file": input_file,
         },
-        volumes=[f"{transformed_directory}:/tmp"],
+        volumes=[f"{os.path.abspath(transformed_directory)}:/tmp"],
```

Comment:
👍 to `os.path.abspath()` here.  I'm unsure why docker containers need an absolute path (sure there is very good reason), but this definitely should satisfy it.